docs/reference/ImageDraw.rst

```diff
@@ -274,6 +274,16 @@ Methods
 
                       .. versionadded:: 4.2.0
 
+    :param language: Language of the text. Different languages may use 
```

Comment:
Move this section after `features`

Reply:
dang, thought I got all of them. anyways, done! thanks again.